databases/core.py

```diff
@@ -489,3 +531,20 @@ def __repr__(self) -> str:
 
     def __eq__(self, other: typing.Any) -> bool:
         return str(self) == str(other)
+
+
+class FakeExecutionContext:
```

Comment:
```suggestion
class DummyExecutionContext:
```
(it's not completely fake, after all)